$NetBSD: patch-aa,v 1.1 2011/10/11 14:53:34 jmmv Exp $

Pull up of http://code.google.com/p/kyua/source/detail?r=230

This fixes the 'configure' stage in, at least, NetBSD/macppc, by preventing
an infinite loop that does not allow the package to build.

--- configure.orig	2011-10-11 14:42:18.000000000 +0000
+++ configure
@@ -3861,8 +3861,7 @@ else
 static void
 first_pass(void)
 {
-    int argc, flag;
-    char ch;
+    int argc, ch, flag;
     char* argv[5];
 
     argc = 4;
@@ -3890,8 +3889,7 @@ first_pass(void)
 static void
 second_pass(void)
 {
-    int argc, flag;
-    char ch;
+    int argc, ch, flag;
     char* argv[5];
 
     argc = 4;
